gradle/dependencies.gradle

```diff
@@ -50,10 +50,10 @@ versions += [
   apacheda: "1.0.0",
   apacheds: "2.0.0-M24",
   argparse4j: "0.7.0",
-  bcpkix: "1.57",
+  bcpkix: "1.58",
   easymock: "3.4",
   jackson: "2.8.5",
-  jetty: "9.2.15.v20160210",
+  jetty: "9.2.22.v20170606",
```

Comment:
@xvrl any concerns with upgrading to this Jetty version? You mentioned previously some issues with certain versions. cc @ewencp

Reply:
@ijuma My info is a bit dated. I don't have much info on the most recent versions of Jetty 9.2.x. The projects I know of have moved on to Java 8 and Jetty 9.3.x.

Reply:
A lot of bugs have probably been fixed so I would upgrade and rollback later if necessary.

Reply:
Thanks @xvrl